apps/network-alerts/js/notification/notification.js

```diff
@@ -18,12 +18,17 @@ function onNotification(message) {
     return;
   }
 
-  var title = Utils.parseParams(message.imageURL).titleID;
+  var title = null;
+  if (message.data) {
+    title = message.data.title;
+  }
+  else {
+    title = Utils.parseParams(message.imageURL).titleID;
```

Comment:
Hmm, interesting, it should not! :) If something is failing, we need to fix it :)